AdobeStockAsset/Model/ResourceModel/Keyword.php

```diff
@@ -33,25 +34,34 @@ class Keyword
      */
     private $keywordFactory;
 
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
```

Comment:
I don't see logger usages in this file, why should it be added?

Reply:
Oh, I missed it. This is a mistakenly adde stuff. Will clean up it in a hour.